platform/android/export/export_plugin.cpp

```diff
@@ -3159,6 +3153,18 @@ Error EditorExportPlatformAndroid::export_project_helper(const Ref<EditorExportP
 		// Write an AndroidManifest.xml file into the Gradle project directory.
 		_write_tmp_manifest(p_preset, p_give_internet, p_debug);
 
+		// Enable or disable Monochrome icon in Gradle build.
+		String icon_xml = "<?xml version=\"1.0\" encoding=\"utf-8\"?>\n"
```

Comment:
We can store this string as a constant, use a placeholder for the section to be updated and use `vformat` to generate the final content.
We should also add a note to ensure that it always matches the content of the `icon.xml` file.

Reply:
@m4gr3d I’ve updated the PR. Please check if this aligns with what you had in mind.